src/common-ssh/guac_ssh.h

```diff
@@ -101,6 +101,33 @@ guac_common_ssh_session* guac_common_ssh_create_session(guac_client* client,
         const char* hostname, const char* port, guac_common_ssh_user* user);
 
 /**
+ * Used within VNC and RDP connections to create an SFTP connection using the
```

Comment:
Dictating exactly where this function is used within the documentation of the function seems questionable to me. Can this not instead be described at a high level, strictly from the perspective of the common SSH client implemented here?